sklearn/ensemble/_forest.py

```diff
@@ -357,11 +357,9 @@ def fit(self, X, y, sample_weight=None):
         y = np.atleast_1d(y)
         if y.ndim == 2 and y.shape[1] == 1:
             warn(
-                (
-                    "A column-vector y was passed when a 1d array was"
-                    " expected. Please change the shape of y to "
-                    "(n_samples,), for example using ravel()."
-                ),
+                "A column-vector y was passed when a 1d array was"
```

Comment:
A few months ago we updated to `black==23.3.0`. May you update to `black==23.3.0` and run the linter again?